### PR TITLE
AD-173 - db refactor: improve how dashboard configuration is stored for teams

### PR DESCRIPTION
**Pull request by rsoaresd**
# Description
- add a new table Configuration composed by Jira config and Bug SLOs config
- support custom team jira queries
- support custom metric for the Jira Bugs affecting CI for each team

## Issue ticket number and link
[AD-173](https://issues.redhat.com/browse/AD-173)
[AD-172](https://issues.redhat.com/browse/AD-172)

### CODE REVIEW

File: backend/api/apis/configuration/v1alpha1/configuration.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package v1alpha1
+
+import (
+	"github.com/google/uuid"
+)
+
+type Configuration struct {
```
Comment (flacatus) [inline backend/api/apis/configuration/v1alpha1/configuration.go:7]: can you comment indicating what mean every member of struct

Comment (rsoaresd) [reply backend/api/apis/configuration/v1alpha1/configuration.go:7]: Sure! Goinggg

**Comment by rsoaresd** [comment]
/retest

outage in production: https://issues.redhat.com/browse/SRVKP-5770

**Pull request merged**